### PR TITLE
Fully reverts the AI revamp

### PR DESCRIPTION
**Pull request by Absolucy**
# Document the changes in your pull request

This _completely reverts_ the AI revamp.

It's consistently buggy as hell, and doesn't really add much except useless complexity. 

Nobody ever builds the AI anything unless it's a calm, peaceful round, and even then it rarely happens. Most AIs don't even bother trying to convince science to build them anything, because they probably won't anyways.

It was a good concept, but it just didn't work (both figuratively and literally).

# Wiki Documentation

Just change it back to the old documentation.

# Changelog

:cl: Lucy
rscdel: AI's back to normal. Go home, everyone.
/:cl:

### CODE REVIEW

**Comment by maxion12345** [comment]
cant thumbs this down so heres a jpeg I got off the internet
emoji_thumbs-down_thumb

**Comment by adamsong** [comment]
Improve don't remove

**Comment by Cartlord** [comment]
I agree with this, my main grievance is that Bibby seems to not actually believe anything is wrong with his ideas - so while he fixes bugs, he refuses to change anything that's "working as intended", even if the way it's intended to work is stupid

**Comment by Runian** [comment]
Should revert unless https://github.com/yogstation13/Yogstation/pull/13311 gets done soon, certain features being re-added, and more bugs get ironed out.

**Comment by maxion12345** [comment]
The AI Revamp is a pretty interesting and novel concept for AI, and I think its a nice refresher on what used to be a pretty boring job playing AI. The datacore/server mechanics are far more interesting aswell, with main benefits being you actually need to construct server rooms to move the core, making it far more difficult for a bad AI to hide. Where before I would unlock my core and have a borg wall me in by some solars, now a borg needs to construct datacores in cold areas like tcomms, xenobio kill room, or build their own cold room. 

tl;dr
I think this AI change is for the better. The research gameplay is certainly intriguing, and having a unique AI will differentiate our server from "every other /tg/ downstream ever" without having to go the route of degeneracy dogborgs and erp

that being said bibby does seem to iron out urgent bugs fast. Might just be a bit burnt-out from yogstation and not actively working on new features.

**Comment by Hopekz** [comment]
> he refuses to change anything that's "working as intended," even if the way it's intended to work is stupid

Literally, anyone can make change PRs to the AI.
@TheGamerdk isn't forced to make all the changes to the AI forever because he was the original author of the rework.
